### DISCUSSION TITLE
[Feature Request/question] HLS Streaming

### DISCUSSION THREAD

**Discussion by jyftw** [remotion-dev/remotion#464] [General]
Hi,

I would like to ask if remotion/player supports HLS streaming. If not, are there any plans to implement this feature anytime soon?

Thanks for taking the time to read, keep up the great work guys!

**Comment by JonnyBurger** [comment]
Hi @jyftw! Thanks for asking!

The question makes me think you have a small misconception. HLS is a protocol for streaming already encoded videos. The Remotion player only plays content written with technologies, so it renders all elements to the DOM instead of playing a real video.

You can however use Remotion to turn the content into a real encoded video and stream that via HLS. But that's not the usecase of @remotion/player.

Hope that explains it well!

**Comment by khanzzirfan** [comment]
Hey @JonnyBurger , There could be a usecase like a Canva.com app 
Consider a scenario of small videos to cut in to Tiktok or Insta videos. I want to make live video editing app, using Remotion in React, I would want to stream only the specific chunks of video when user does all the editing and all customization with emojis/stickers. So rendering full length video is too large for video editing , not a good user experience.  Hence users video trimmed into hls on the backend at the same time as use edits and stream the chunks of the video content.

**Reply by JonnyBurger** [reply]
@khanzzirfan We are working in that direction to only edit the part of the video that is necessary, follow this issue: #1292
